app/components/primer/task_list_item.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <label class="text-normal">
```

Comment:
Mind adding a comment about accessibility catch to the PR?

Reply:
@khiga8; the accessibility CI check caught me trying to use a `<input>` without a `<label>` which I then added. It's already paying off! 🎉